### DISCUSSION TITLE
Tabs not disappearing after removing beatmaps

### DISCUSSION THREAD

**Discussion by Jan-Arbus** [ppy/osu#21273] [Q&A] [answered]
https://user-images.githubusercontent.com/74862110/202432269-b3044c26-d39d-4cf0-89d1-fcff56df09b0.mp4

The beatmap is actually removed, but the Tabs are still there
[network.log](https://github.com/ppy/osu/files/10030899/network.log)
[database.log](https://github.com/ppy/osu/files/10030900/database.log)
[performance.log](https://github.com/ppy/osu/files/10030901/performance.log)
[runtime.log](https://github.com/ppy/osu/files/10030902/runtime.log)

**Comment by Loreos7** [comment, marked as answer]
Dublicate of https://github.com/ppy/osu/issues/21272
